collections/_article/dev-snapshot-godot-4-1-beta-3.md

```diff
@@ -0,0 +1,109 @@
+- GUI: Fix position and size of the `MenuButton` popup ([GH-78269](https://github.com/godotengine/godot/pull/78269)).
+- GUI: Fix wrong order in auto-completion sorting with empty string to complete ([GH-78321](https://github.com/godotengine/godot/pull/78321)).
+- Import: Fix Basis Universal compressing with normal maps ([GH-62718](https://github.com/godotengine/godot/pull/62718)).
+- Import: Respect `compress_binary_resources` editor setting on scene import ([GH-76630](https://github.com/godotengine/godot/pull/76630)).
+- Input: Fix `OS.find_scancode_from_string` not working with modifiers ([GH-47996](https://github.com/godotengine/godot/pull/47996)).
+- Input: Fix clearing custom cursor ([GH-74511](https://github.com/godotengine/godot/pull/74511)).
+- Navigation: Add agent pause mode to `NavigationServer` ([GH-75888](https://github.com/godotengine/godot/pull/75888)).
+- Navigation: Fix `NavigationAgent2D` path debug hidden behind canvas items ([GH-78438](https://github.com/godotengine/godot/pull/78438)).
+- Network: Allow `ENetConnection` to send a packet to an arbitrary destination ([GH-77627](https://github.com/godotengine/godot/pull/77627)).
+- Physics: Improve rigid body CCD against moving bodies ([GH-76138](https://github.com/godotengine/godot/pull/76138)).
+- Plugin: Add relative path support for `EditorPlugin.add_autoload_singleton` ([GH-78109](https://github.com/godotengine/godot/pull/78109)).
+- Porting: Fix `bsd` feature tag includes only "other BSDs" ([GH-78272](https://github.com/godotengine/godot/pull/78272)).
+- Porting: Android: Allow exporting release builds without a debug keystore ([GH-78123](https://github.com/godotengine/godot/pull/78123)).
+- Porting: Linux/BSD: Fix temporary file permissions ([GH-78347](https://github.com/godotengine/godot/pull/78347)).
+- Rendering: Add RENDERING_INFO parameters to GL Compatibility renderer ([GH-77536](https://github.com/godotengine/godot/pull/77536)).
+- Rendering: Ensure filter/repeat is cached by Viewport texture in GL Compatibility renderer ([GH-78285](https://github.com/godotengine/godot/pull/78285)).
+- Rendering: Copy texture filter/repeat modes when replacing a texture in the GL Compatibility backend ([GH-78287](https://github.com/godotengine/godot/pull/78287)).
+- Rendering: Fix invalid RID errors when freeing a mesh with blend shapes ([GH-78433](https://github.com/godotengine/godot/pull/78433)).
+- Rendering: Fix volumetric fog in stereo by projection vertex in combined space ([GH-78436](https://github.com/godotengine/godot/pull/78436)).
+- Tilemaps: Fix crashes when painting a large area in a tilemap ([GH-76548](https://github.com/godotengine/godot/pull/76548)).
+- Tilemaps: Fix TileMap draw preview when switching CanvasItem editor tool ([GH-78467](https://github.com/godotengine/godot/pull/78467)).
+- Tilemaps: Fix TileSet set as local to scene ([GH-78477](https://github.com/godotengine/godot/pull/78477)).
```

Comment:
Should "Tilemaps" be a top level category, or go in "2D: Tilemaps:"?

Reply:
I used it as a top category at some point when there was a lot of stuff to report about tilemaps. But I can double prefix it this time.